src/pipeline_migration/actions/add_task.py

```diff
@@ -0,0 +1,406 @@
+import argparse
+import logging
+import re
+import subprocess
+from argparse import ArgumentTypeError
+from collections.abc import Generator, Iterable
+from pathlib import Path
+from typing import Any, Final
+
+import oras.defaults
+import requests
+from packaging.version import parse as parse_version
+
+from pipeline_migration.actions.migrate import NotAPipelineFile, resolve_pipeline
+from pipeline_migration.quay import get_active_tag
+from pipeline_migration.registry import REGISTRY, Container
+from pipeline_migration.types import FilePath
+from pipeline_migration.utils import load_yaml, dump_yaml, YAMLStyle
+
+logger = logging.getLogger("add_task")
+
+
+SUBCMD_DESCRIPTION: Final = """\
+The following are several examples with a Konflux task push-dockerfile:
+
+* Add task with latest bundle to pipelines within relative .tekton/ directory.
+
+    cd /path/to/repo
+    pipeline-migration-tool add-task push-dockerfile
+
+* Add task to mulitple pipelines in several repositories:
+
+    pipeline-migration-tool add-task push-dockerfile \\
+        /path/to/repo1/.tekton/pr.yaml /path/to/repo2/.tekton/push.yaml
+
+* Add task with parameter and execution order:
+
+    pipeline-migration-tool add-task push-dockerfile \\
+        --param param1,value1 --param param2,value2 \\
```

Comment:
For param-value pairs, IMO it would feel more natural to use `--param param1=value1`

That's what `tkn` uses as well
```
tkn pipeline start -f docker-build-oci-ta.yaml --use-param-defaults \
  -p git-url=https://github.com/chmeliik/checkton \
  -p output-image=quay.io/acmiel-test/test
```

Reply:
Updated. PTAL.